store/src/java-test/com/zimbra/cs/service/mail/ItemActionTest.java

```diff
@@ -116,9 +116,11 @@ public void copyMessageFromDraftsToSent() throws Exception {
         ItemId iidTarget = new ItemId(mbox, Mailbox.ID_FOLDER_SENT);
         ItemActionHelper op = ItemActionHelper.COPY(null, mbox, SoapProtocol.Soap12, Arrays.asList(msgId), MailItem.Type.MESSAGE, null, iidTarget);
         Assert.assertNotNull(op);
-        Assert.assertNotNull(op.getResult().getSuccessIds());
-        Assert.assertEquals(1, op.getResult().getSuccessIds().size());
-        ItemId iid = new ItemId(op.getResult().getSuccessIds().get(0), acct.getId());
+        Assert.assertTrue(op.getResult() instanceof CopyActionResult);
```

Comment:
Again.  It is nice to be able to look at a test report and get a feel for what is failing instantly rather than having to drill down into the test source to find out.